examples/ChimneySwap/app/src/main/java/xyz/thepathfinder/www/chimneyswap/Chimney.java

```diff
@@ -0,0 +1,74 @@
+package xyz.thepathfinder.www.chimneyswap;
+
+import android.graphics.Bitmap;
+
+import com.google.android.gms.maps.model.LatLng;
+import com.google.gson.JsonObject;
+
+public class Chimney {
+
+    private int id;
+    private String name;
+    private String imageUrl;
+    private Bitmap image;
+    private LatLng position;
+
+    public Chimney() {
+        this.id = 0;
```

Comment:
No the Java's compiler is smart enough to remove redundant initialization. Anyway, it is better to not leave ambiguity in the code.